packages/block-editor/src/components/inserter/index.js

```diff
@@ -19,6 +19,33 @@ import { plusCircle } from '@wordpress/icons';
  */
 import InserterMenu from './menu';
 
+const additionalStyles = [
```

Comment:
That additional styles are added to show how to pass styles for `hover`, `focus` etc to the` PrimitiveButton`